### PR TITLE
Preprintedits

### PR DESCRIPTION
**Pull request by emhart**
I've made some minor edits to the draft before pushing to a pre-print server.

### CODE REVIEW

File: manuscript/manuscript.md [hunk 1]
```diff
@@ -491,6 +496,9 @@ of interest as outlined in Rule 9.
   it should be impossible to recover the message from the output, and any
   modifications to the message should also modify the output.
 
+* Apache **Spark** is an open source computing platform for querying large data
+  sets in memory, in contrast to on disk based methods like MapReduce
```
Comment (naupaka) [inline manuscript/manuscript.md]: Add period at end of definition

File: manuscript/manuscript.md [hunk 2]
```diff
@@ -467,11 +466,14 @@ of interest as outlined in Rule 9.
 * **HDF5** (Hierarchical Data Format) is an open-source binary file format designed
   to store large amount of data (and their associated metadata) by providing a
   hierarchical structure that could be compared to how a hard drive is organized
-  with directories and files.
+  with directories and files. It is maintained by the non-profit HDF Group, a
+  spin off of the National Center for Supercomputing Applications (NCSA)
```
Comment (naupaka) [inline manuscript/manuscript.md]: Add period at end of definition

File: manuscript/manuscript.md [hunk 3]
```diff
@@ -313,20 +310,22 @@ collaborators, and yourself. Both the NSF and NIH have data sharing policies in
...
+quickly recognized that the cityanonymized the data with a simple MD5 hashing
+scheme and all 20 GB of data was de-anonymized in a matter of hours [@Goodin].
+
+In more problematic cases, the data itself allows identifiability: this is the
+case with human genomic data that map directly onto a subject's identity [@Homer2008].
+Methods for dealing with these complex issues at the intersection of data storage
+and privacy are rapidly evolving, and include storing changes against a reference
+genome to help with privacy and data volume [@Kahn2011; @wandelt2014trends], or
+bringing computation to data storage facilities instead of vice versa [@Gaye2014].
+Havinga plan for privacy before data acquisition is important, because it can
```
Comment (naupaka) [inline manuscript/manuscript.md]: "Havinga" -> "Having a"

File: manuscript/manuscript.md [hunk 4]
```diff
@@ -528,7 +536,7 @@ of interest as outlined in Rule 9.
 National Center for Supercomputing Applications.
 Software Carpentry Foundation.
 iDigBio/NSF.
```
Comment (naupaka) [inline manuscript/manuscript.md]: Please add my funding info in: "NBZ was funded by the Gordon and Betty Moore Foundation through Grant GBMF 2550.03 to the Life Sciences Research Foundation."

Comment (jhollist) [inline manuscript/manuscript.md]: And I did find out that we should include some EPA verbiage.  So please include:

> Contributions from JWH, an employee of the U.S. Environmental Protection Agency (EPA)
> have not been subjected to the Agency's peer and administrative review. Therefore, the
> conclusions and opinions drawn are solely those of the author(s) and are not necessarily
> the views of the Agency.  Mention of trade names or commercial products does not
> constitute endorsement or recommendation for use.

**Comment by fmichonneau** [comment]
Could you please add the following in the acknowledgements for me?

> FM was funded by iDigBio (Integrated Digitized Biocollections), and therefore this material is based upon work supported by the National Science Foundation’s Advancing Digitization of Biodiversity Collections Program (Cooperative Agreement EF-1115210).

**Comment by tpoisot** [comment]
And for me:

> TP was funded by a NSERC Discovery grant, and a Start-Up grant from the Université de Montréal.

**Comment by tpoisot** [comment]
@emhart I would suggest that we use !biorxiv, since they won't let us modify the preprint after it is accepted. PeerJ?

**Pull request merged**